test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -5711,5 +5711,220 @@ public void Dotnet_New_Template_Restore_Pack_Success(string template)
                 Assert.True(File.Exists(nupkgPath));
             }
         }
+
+        [Theory]
+        [InlineData("netstandard2.1")]
+        [InlineData("net5.0")]
+        public async Task PackCommand_PackProject_PackageReference_PreReleaseDependency_Fails(string platform)
+        {
+            // Arrange
+            using (var pathContext = msbuildFixture.CreateSimpleTestPathContext())
+            {
+                var stableDependencyName = "StablePackage";
+                var stableDependencyVersion = "2.0.0";
+                var stableDependencyPackage = new SimpleTestPackageContext(stableDependencyName, stableDependencyVersion);
+                stableDependencyPackage.Files.Clear();
+                stableDependencyPackage.AddFile("_._");
```

Comment:
What's the intent for this stable package?  To me it looks unrelated to the scenario defined by the test method name, and just makes the test several lines longer.

Reply:
Good point, removed.